### PR TITLE
Remove pull-reques:create --issue option

### LINKED ISSUE #418: [RFC] pull-reques:create remove --issue option
The `--issue` only works for GitHub and I don't really see it being used that often (anymore), and to me it also feels a bit strange to replace an existing issue with a PR keeping all the original comments :smile: 

I already asked to remove this in #166 but @cordoval said to use it :)
@gushphp/contributor what do you think about removing this option?

I don't have problem with keeping this option, but pull-request:create needs to be rewritten as the complexity is way to high: https://scrutinizer-ci.com/g/gushphp/gush/code-structure/master/operation/Gush%5CCommand%5CPullRequest%5CPullRequestCreateCommand%3A%3Aexecute

### PR DESCRIPTION
**Pull request by sstok**
closes #418

### CODE REVIEW

File: tests/Command/PullRequest/PullRequestCreateCommandTest.php [hunk 1]
```diff
@@ -132,7 +97,7 @@ public function opens_pull_request_to_a_specific_organization_or_username($args)
         $tester->execute($args, ['interactive' => false]);
 
         $res = trim($tester->getDisplay(true));
-        $this->assertContains('Making PR from '.$args['--source-org'].':issue-145 to gushphp:master', $res);
+        $this->assertContains('Opened pull request', $res);
```
Comment (cordoval) [inline tests/Command/PullRequest/PullRequestCreateCommandTest.php]: maybe some kind of feedback? or is this output elsewhere?

Comment (sstok) [inline tests/Command/PullRequest/PullRequestCreateCommandTest.php]: https://github.com/sstok/gush/blob/418-pull-reques-create-remove-issue-option/src/Command/PullRequest/PullRequestCreateCommand.php#L152-L158

https://github.com/sstok/gush/blob/418-pull-reques-create-remove-issue-option/tests/Command/PullRequest/PullRequestCreateCommandTest.php#L50-L51

 ;) its already shown in normal mode.

Comment (cordoval) [inline tests/Command/PullRequest/PullRequestCreateCommandTest.php]: so we have a link already?

Comment (sstok) [inline tests/Command/PullRequest/PullRequestCreateCommandTest.php]: The link is still there :) I amended the commit to include the link in the expected output.
This was only to replace the expectation as you need at least one expectation.

**Pull request merged**